src/Sylius/Component/Core/Uploader/ImageUploader.php

```diff
@@ -99,6 +101,7 @@ protected function sanitizeContent(string $fileContent, string $mimeType): strin
     {
         if (self::MIME_SVG_XML === $mimeType || self::MIME_SVG === $mimeType) {
             $fileContent = $this->sanitizer->sanitize($fileContent);
+            Assert::notFalse($fileContent);
```

Comment:
This asserts looks like it might actually be hit by end-users. What do you thing adding error message as this assert will be un-debugable for them otherwise.

Something like
```suggestion
            Assert::notFalse($fileContent, sprintf('Failed to sanitize uploaded "%s" image', $mimeType));
```